provisioning/transport/amqp/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "azure-iot-provisioning-amqp",
+  "version": "0.0.1",
+  "description": "AMQP transport for Azure Device Provisioning SDK",
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "main": "index.js",
+  "typings": "index.d.ts",
+  "dependencies": {
+    "azure-iot-amqp-base": "1.2.0",
+    "azure-iot-common": "1.2.0",
+    "debug": "^3.0.1",
+    "machina": "^2.0.0"
+  },
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "istanbul": "^0.4.4",
+    "jshint": "^2.9.2",
+    "mocha": "^3.0.1",
+    "sinon": "^1.17.5",
+    "tslint": "^5.1.0",
+    "typescript": "2.2.2",
+    "@types/node": "^7.0.5"
+  },
```

Comment:
I'd keep these as empty as possible, it's easy to forget to remove unused dependencies.